<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNotepadClass</class>
 <widget class="QMainWindow" name="QNotepadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNotepad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPageSetup"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindNext"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGo"/>
    <addaction name="separator"/>
    <addaction name="actionAll"/>
    <addaction name="actionDate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionPageSetup">
   <property name="text">
    <string>页面设置</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>查找下一个</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="text">
    <string>转到</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="text">
    <string>时间/日期</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QNotepad.qrc"/>
 </resources>
 <connections/>
</ui>
